### PR TITLE
Add Event Command test cases

### PR DESCRIPTION
**Pull request by vijay-shankaranand**

### CODE REVIEW

File: src/main/java/seedu/address/model/event/Event.java [hunk 1]
```diff
@@ -19,11 +21,12 @@
 public class Event {
 
     // Identity fields
-    private final EventName name;
+    private final Name name;
```
Comment (Khaleelur-Rahman) [inline src/main/java/seedu/address/model/event/Event.java:24]: Similarly over here, I feel having eventName would be better.

**Review by Khaleelur-Rahman — APPROVED** [review]
LGTM! Great initiative to abstract out the Name and Address classes, since these are repeated across multiple classes. Just a few naming nits to ponder upon :)

File: src/main/java/seedu/address/model/event/Event.java [hunk 2]
```diff
@@ -19,11 +21,12 @@
...
 
     // Data fields
+    private final Address address;
     private final Date date;
```
[the remark this answers is not in the document]
Comment (vijay-shankaranand) [reply src/main/java/seedu/address/model/event/Event.java]: fixed

File: src/main/java/seedu/address/logic/commands/event/LinkCommand.java [hunk 3]
```diff
@@ -39,14 +38,14 @@ public class LinkCommand extends Command {
...
+    private final Name nameToLink;
     private final Set<Name> contactNameListToLink;
 
     /**
      * Creates a LinkCommand to link the person specified by the name to the event specified by the name.
      */
-    public LinkCommand(EventName eventNameToLink, Set<Name> contactNameListToLink) {
-        this.eventNameToLink = eventNameToLink;
+    public LinkCommand(Name nameToLink, Set<Name> contactNameListToLink) {
+        this.nameToLink = nameToLink;
```
[the remark this answers is not in the document]
Comment (vijay-shankaranand) [reply src/main/java/seedu/address/logic/commands/event/LinkCommand.java]: fixed

**Pull request merged**